### PR TITLE
Add Travis_CI

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

**Comment by mprahl** [comment]
Can you use my TravisCI for MBS-UI as inspiration?
https://github.com/release-engineering/mbs-ui/commit/dccfb3a19afbae88ca271a44631938a09ccbe2da

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+language: node_js
+node_js:
+  - 9
+
+cache:
+  directories:
+    - "node_modules"
+
+branches:
```
Comment (mprahl) [inline .travis.yml]: This block can be removed

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+before_script:
+  - export CHROME_BIN=chromium-browser
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+install:
+- npm install
+
+script:
+- npm run ci
```
Comment (mprahl) [inline .travis.yml]: Add a new line at the end of the file please

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+
+language: node_js
+node_js:
+  - 9
+
+cache:
```
Comment (mprahl) [inline .travis.yml]: Cool!

**Comment by mprahl** [comment]
Some small changes, then +1

**Comment by mprahl** [comment]
@yashvardhannanavati can you fix your git authorship and rebase on master?

**Comment by yashvardhannanavati** [comment]
@mprahl Sure, done! :smiley:

**Comment by mprahl** [comment]
@yashvardhannanavati looks like your last push didn't address the comments. Can you push again?

**Comment by yashvardhannanavati** [comment]
Oops! sorry about that

**Pull request merged**